docgen/src/stylesheets/components/_code-highlighting.scss

```diff
@@ -7,41 +7,111 @@
-.cm-s-mdn-like .cm-header { color: #FF6400; }
-.cm-s-mdn-like .cm-hr { color: #AEAEAE; }
-.cm-s-mdn-like .cm-link { color:#ad9361; font-style:italic; text-decoration:none; }
-.cm-s-mdn-like .cm-error { border-bottom: 1px solid red; }
-
-div.cm-s-mdn-like .CodeMirror-activeline-background { background: #efefff; }
-div.cm-s-mdn-like span.CodeMirror-matchingbracket { outline:1px solid grey; color: inherit; }
-
-.cm-s-mdn-like .cm-positive {color: #690; background: #efe; }
-.cm-s-mdn-like .cm-negative {color: red; background: #fee}
+// sass-lint:disable class-name-format, force-pseudo-nesting, no-vendor-prefixes
+.cm-s-mdn-like.CodeMirror {
+  color: #999;
+  background-color: #FFF;
+}
+.cm-s-mdn-like .CodeMirror-selected {
+  background: #CFC;
+}
+.cm-s-mdn-like .CodeMirror-line::selection, .cm-s-mdn-like .CodeMirror-line > span::selection, .cm-s-mdn-like .CodeMirror-line > span > span::selection {
+  background: #CFC;
+}
+.cm-s-mdn-like .CodeMirror-line::-moz-selection, .cm-s-mdn-like .CodeMirror-line > span::-moz-selection, .cm-s-mdn-like .CodeMirror-line > span > span::-moz-selection {
+  background: #CFC;
+}
+.cm-s-mdn-like .CodeMirror-gutters {
+  background: #F8F8F8;
+  border-left: 6px solid rgba(0, 83, 159, 0.65);
+  color: #333;
+}
+.cm-s-mdn-like .CodeMirror-linenumber {
+  color: #AAA;
+  padding-left: 8px;
+}
+.cm-s-mdn-like .CodeMirror-cursor {
+  border-left: 2px solid #222;
+}
+.cm-s-mdn-like .cm-keyword {
+  color: #6262FF;
+}
+.cm-s-mdn-like .cm-atom {
+  color: #F90;
+}
+.cm-s-mdn-like .cm-number {
+  color: #CA7841;
+}
+.cm-s-mdn-like .cm-def {
+  color: #8DA6CE;
+}
+.cm-s-mdn-like .cm-variable-2, .cm-s-mdn-like .cm-tag {
+  color: #690;
+}
+.cm-s-mdn-like .cm-variable-3, .cm-s-mdn-like .cm-def, .cm-s-mdn-like .cm-type {
+  color: #07A;
+}
+.cm-s-mdn-like .cm-variable {
+  color: #07A;
+}
+.cm-s-mdn-like .cm-property {
+  color: #905;
+}
+.cm-s-mdn-like .cm-qualifier {
+  color: #690;
+}
+.cm-s-mdn-like .cm-operator {
+  color: #CDA869;
+}
+.cm-s-mdn-like .cm-comment {
+  color: #777;
+  font-weight: normal;
+}
+.cm-s-mdn-like .cm-string {
+  color: #07A;
+  font-style: italic;
+}
+.cm-s-mdn-like .cm-string-2 {
+  color: #BD6B18;
+}
+.cm-s-mdn-like .cm-meta {
+  color: #000;
+}
+.cm-s-mdn-like .cm-builtin {
+  color: #9B7536;
+}
+.cm-s-mdn-like .cm-tag {
+  color: #997643;
+}
+.cm-s-mdn-like .cm-attribute {
+  color: #D6BB6D;
+}
+.cm-s-mdn-like .cm-header {
+  color: #FF6400;
+}
+.cm-s-mdn-like .cm-hr {
+  color: #AEAEAE;
+}
+.cm-s-mdn-like .cm-link {
+  color: #AD9361;
+  font-style: italic;
+  text-decoration: none;
+}
+.cm-s-mdn-like .cm-error {
+  border-bottom: 1px solid red;
+}
+.cm-s-mdn-like .CodeMirror-activeline-background {
+  background: #EFEFFF;
+}
+.cm-s-mdn-like .CodeMirror-matchingbracket {
+  outline: 1px solid grey;
+  color: inherit;
+}
+.cm-s-mdn-like .cm-positive {
+  color: #690;
+  background: #EFE;
+}
+.cm-s-mdn-like .cm-negative {
+  color: red;
+  background: #FEE;
+}
```

Comment:
An way to enforce space between selectors with sass-lint? This is not very readable.

Reply:
What do you mean?

Reply:
This:

```

Reply:
This:

```
 .cm-s-mdn-like.CodeMirror {
   color: #999;
   background-color: #FFF;
 }

 .cm-s-mdn-like .CodeMirror-selected {
   background: #CFC;
 }

 .cm-s-mdn-like .CodeMirror-line::selection, .cm-s-mdn-like .CodeMirror-line > span::selection, .cm-s-mdn-like .CodeMirror-line > span > span::selection {
   background: #CFC;
 }

 .cm-s-mdn-like .CodeMirror-line::-moz-selection, .cm-s-mdn-like .CodeMirror-line > span::-moz-selection, .cm-s-mdn-like .CodeMirror-line > span > span::-moz-selection {
   background: #CFC;
 }
```

Instead of this:

```
 .cm-s-mdn-like.CodeMirror {
   color: #999;
   background-color: #FFF;
 }
 .cm-s-mdn-like .CodeMirror-selected {
   background: #CFC;
 }
 .cm-s-mdn-like .CodeMirror-line::selection, .cm-s-mdn-like .CodeMirror-line > span::selection, .cm-s-mdn-like .CodeMirror-line > span > span::selection {
   background: #CFC;
 }
 .cm-s-mdn-like .CodeMirror-line::-moz-selection, .cm-s-mdn-like .CodeMirror-line > span::-moz-selection, .cm-s-mdn-like .CodeMirror-line > span > span::-moz-selection {
   background: #CFC;
 }
```

Reply:
https://github.com/sasstools/sass-lint/blob/develop/docs/rules/empty-line-between-blocks.md, right @ronanlevesque

Reply:
Exactly!

Reply:
Ok I will add that